assets/js/common/ModifiedCheckPill/ModifiedCheckPill.test.jsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import { screen, render } from '@testing-library/react';
+import '@testing-library/jest-dom';
+
+import ModifiedCheckPill from './ModifiedCheckPill';
+
+describe('ModifiedCheckPill component', () => {
+  it('should not render modified pill for a non customized check', () => {
+    render(<ModifiedCheckPill customized={false} />);
+
+    expect(screen.queryByText('MODIFIED')).toBeNull();
```

Comment:
if you consider to add the label prop, we could test for default value and changed value